BoxContentSDK/BoxContentSDK/BOXContentSDKErrors.m

```diff
@@ -16,7 +16,7 @@
 NSString *const BOXAuthTokenRequestErrorInvalidRequest = @"invalid_request";
 
 NSString *const BOXAuthErrorUnauthorizedDevice = @"unauthorized_device";
-NSString *const BOXAuthErrorExceededDeviceLimit = @"exceeded_device_limit";
+NSString *const BOXAuthErrorExceededDeviceLimit = @"exceeded_device_limits";
```

Comment:
I'm pretty sure this error code is deprecated as I've not been able to get it from testing, I'll reach out to the API team to confirm. However, I would follow the documentation and not what was in Bender.